SU2_CFD/src/solver_direct_mean.cpp

```diff
@@ -2897,15 +2894,15 @@ void CEulerSolver::SetInitialCondition(CGeometry **geometry, CSolver ***solver_c
           
           if (Distance < Radius) {
             
-            solver_container[iMesh][FLOW_SOL]->node[iPoint]->SetSolution(0, Density_CylND);
+            solver_container[iMesh][FLOW_SOL]->node->SetSolution(iPoint, 0, Density_CylND);
```

Comment:
`snode` is only used inside the solver (i.e. on uses equivalent to `this->snode`) if we are manipulating the FLOW_SOL position of the solver_container we still need to access `node` (well technically we could static cast but that would be ugly).

Reply:
oh, i guess that is the answer to the question I asked above. nvm.